packages/fuels-signers/src/lib.rs

```diff
@@ -268,15 +275,54 @@ mod tests {
 
         // Assert that we've sent 2 from wallet 1, resulting in an amount of 3 in wallet 1.
         let resulting_amount = wallet_1_final_coins.first().unwrap();
-        assert_eq!(resulting_amount.amount(), 3);
+        assert_eq!(resulting_amount.amount(), AMOUNT - SEND_AMOUNT);
 
         let wallet_2_final_coins = wallet_2.get_coins(BASE_ASSET_ID).await?;
-        assert_eq!(wallet_2_final_coins.len(), 2);
+        assert_eq!(wallet_2_final_coins.len(), 1);
 
         // Check that wallet 2's amount is 7:
         // 5 initial + 2 that was sent to it.
```

Comment:
These comments are not valid anymore. Maybe delete all the comments as it is clear from the code what we are doing...